unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Unpublish/UnpublishPopupController.cs

```diff
@@ -31,7 +31,7 @@ public void Dispose()
         CoroutineStarter.Stop(fakeProgressRoutine);
         view.OnCancelPressed -= OnCancel;
         view.OnConfirmPressed -= OnConfirmUnpublish;
-        view.Dispose();
+        view?.Dispose();
```

Comment:
```suggestion
        if ( view != null ) view.Dispose();
```
Can we use a normal if here? if view is a unity object the null coalescing operator may fail.